templates/hooks/sdk_addon.go.tpl

```diff
@@ -0,0 +1,24 @@
+{{ $outputShape := .CRD.GetOutputShape .CRD.Ops.Create }}
+// This method copies the data from given {{ $outputShape.ShapeName }} by populating it
+// into copy of supplied resource and returns that.
+func (rm *resourceManager) set{{ $outputShape.ShapeName }}Output (
+    r *resource,
+    obj *svcsdk.{{ $outputShape.ShapeName }},
+) (*resource, error) {
+	if obj == nil ||
+		r == nil ||
+		r.ko == nil {
+		return nil, nil
+	}
+	resp := &svcsdk.{{ .CRD.Ops.Create.OutputRef.Shape.ShapeName }}{ {{ $outputShape.ShapeName }}:obj }
+	// Merge in the information we read from the API call above to the copy of
+    // the original Kubernetes object we passed to the function
+    ko := r.ko.DeepCopy()
```

Comment:
note you've got a couple indentation/spacing issues in this template. `go fmt` is executed on the resulting file, which is why these issues are corrected during `make build-controller`, but you may want to address them here anyway

Reply:
Thanks @jaypipes I have updated the template  and regenerated the code after rebasing `code-generator` and `elasticache-controller` repositories.

`make test` passed.